public/src/components/drawer.tsx

```diff
@@ -199,11 +199,25 @@ export default function NavDrawer(): React.ReactElement {
           href="https://docs.google.com/document/d/15XeUDHMSt7Urv0I3LyAAhC8kcC2JRWtkiD1xlEVitcE"
           key="Testing FAQ"
           className={classes.link}
+          target="_blank"
+          rel="noopener noreferrer"
```

Comment:
Thought whilst I was here I'd update the FAQ link to open in a new tab as that's probably what a user would want